Hydro-Grapher_AWDR_GWSI_v1.12_TRANSDUCERS_20191216.py

```diff
@@ -59,38 +62,39 @@
 wells = list(set(df[col]))   ## Mike G said this is not the best way, use the set method below
 wells = set(df[col])
 
-for location in wells:
-    df3 = pd.merge(df,df2, on="WELL_SITE_ID")
 
-    df4 = df3.loc[df3[col]==location]
+def figure_proc(location):
+    df3 = pd.merge(df, df2, on="WELL_SITE_ID")

```

Comment:
We should probably move this out of the multiprocessing loop. We should merge the dataframes ahead of time and send just a slice of the dataframe to each method to reduce overhead.